sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_service_groundedness/_service_groundedness.py

```diff
@@ -0,0 +1,136 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from typing import Coroutine, Optional, Dict
+from typing_extensions import override
+
+from azure.ai.evaluation._common.constants import EvaluationMetrics
+from azure.ai.evaluation._evaluators._common import RaiServiceEvaluatorBase
+
+
+class GroundednessProEvaluator(RaiServiceEvaluatorBase):
+    """
+    Initialize a Groundedness Pro evaluator for determine if the response is grounded
+    in the query and context.
+
+    If this evaluator is supplied to the `evaluate` function, the aggregated metric
+    for the groundedness pro label will be "groundedness_pro_passing_rate".
+
+    :param credential: The credential for connecting to Azure AI project. Required
+    :type credential: ~azure.core.credentials.TokenCredential
+    :param azure_ai_project: The scope of the Azure AI project.
+        It contains subscription id, resource group, and project name.
+    :type azure_ai_project: ~azure.ai.evaluation.AzureAIProject
+
+    **Usage**
+
+    .. code-block:: python
+
+        azure_ai_project = {
+            "subscription_id": "<subscription_id>",
+            "resource_group_name": "<resource_group_name>",
+            "project_name": "<project_name>",
+        }
+        eval_fn = GroundednessProEvaluator(azure_ai_project)
```

Comment:
need to add a credential as the first parameter here.. though I'm not sure the other code samples have this either

Reply:
Added, will add others in a separate PR